### ISSUE TITLE
TTY text is not included in the link text. (00.00.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#39856]
#### VFS team name
Cedar-1095b

#### VFS product name
1095B Tax Form

#### VFS feature name
1095B Tax Form

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
When download fails and users receive the "Sorry, something went wrong..." message, one of the recommended contact methods is a TTY phone number for deaf or hard-of-hearing users. From the link text alone, the context that it's a TTY number is not clear.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
Include the "TTY" text in the link text and in the associated aria-label, eg. `<a href="tel:+18008778339" aria-label="TTY. 8 0 0. 8 7 7. 8 3 3 9.">TTY: 800-877-8339</a>`.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by JordanSWhite** [comment]
@briandeconinck Do you have any guidance on @stephte's question above?

**Comment by briandeconinck** [comment]
@stephte Sorry for the slow reply, and thank you @JordanSWhite for the nudge!

Yes, a separate link for TTY 711. Note that both should have the aria-label for better announcing by screen readers.

**Comment by stephte** [comment]
ok sounds good, thank you!

**Comment by stephte** [comment]
The PR with the fix was merged in just a moment ago

**Comment by briandeconinck** [comment]
Great, thank you!

**Comment by tameracorson** [comment]
@briandeconinck So, would this make this staging review TTY ticket closed?   I ask since @stephte is unexpectedly rolling onto a different product than the rest of the team and I wanted to tie any lose ends before he departs.  Please advise

**Comment by shiragoodman** [comment]
@tameracorson if the TTY text is now included in the link text, then yes, this ticket can be closed. If you need help validating the fix, please let us know.

cc @briandeconinck

**Comment by allison0034** [comment]
This ticket has been reassigned from CCIssue00.00 "No VA.gov Experience Standard for the issue found" to CCIssue08.03 "Link and button labels". No VFS action is needed.
